test/test_jit.py

```diff
@@ -7551,8 +7552,6 @@ def method(self, x):
 
     def test_script_define_order(self):
         class M(torch.jit.ScriptModule):
-            def __init__(self):
```

Comment:
Why don't we need to run super's init here?

Reply:
If there is no `__init__` then you just inherit the super's `__init__`. If you define your _own_ function, then you must call super. Not the best-designed thing in python...